cmd/support.go

```diff
@@ -38,7 +37,7 @@ var supportCmd = &cobra.Command{
 
 		_, certProjectTypeValue, err := certProjectTypeLabel.Run()
 		if err != nil {
-			return errors.ErrSupportCmdPromptFailed
+			return ErrSupportCmdPromptFailed
```

Comment:
According to the doc included here, we should be writing these errors in-line since they don't control execution, correct?

Reply:
Again, this package has not gotten that treatment yet. The main focus of this particular PR is to be rid of the global "errors" package. #654 begins to address this. There will be others. Trying to get a big PR knocked out, that we can then have subsequent PRs that go package or two at a time, and don't have to rely on each other.